src/main.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom';
+import superagent from 'superagent';
+import './style/main.scss';
+
+const apiUrl = 'https://www.reddit.com/r';
+
+class RedditSearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      thingSearchedFor: '',
+      searchLimit: '',
+    };
+
+    // this.handlePokemonNameChange = this.handlePokemonNameChange.bind(this);
+    this.searchFormBoard = this.searchFormBoard.bind(this);
+    // this.searchFormLimit = this.searchFormLimit.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  searchFormBoard(event) {
+    this.setState({ thingSearchedFor: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.redditSelect(this.state.thingSearchedFor); // eslint-disable-line
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <input
+          type="text"
+          name="thing searched for"
+          placeholder="Search for a Reddit page"
+          value={this.state.thingSearchedFor}
+          onChange={this.searchFormBoard}
+        />
+      </form>
+    );
+  }
+}
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      redditLookUp: {},
+      thingSearchedFor: null,
+      redditNameError: null,
+      searchLimit: null,
+    };
+
+    this.redditSelect = this.redditSelect.bind(this);
+    this.renderSearchResults = this.renderSearchResults.bind(this);
+  }
+
+  redditSelect(thingSearchedFor) {
```

Comment:
take in a second parameter for limit and adjust api path with dynamically rendered limit, just like you have for the topic